site/src/pages/TerminalPage/TerminalPage.tsx

```diff
@@ -134,13 +127,13 @@ const TerminalPage: FC = () => {
     window.addEventListener("resize", listener);
 
     // Terminal is correctly sized and is ready to be used.
-    setTerminal(terminal);
+    terminalRef.current = terminal;
 
     return () => {
       window.removeEventListener("resize", listener);
       terminal.dispose();
     };
-  }, [theme, renderer, config.isLoading, xtermRef, handleWebLinkRef]);
+  }, [theme, renderer, config.isLoading, terminalWrapperRef, handleWebLinkRef]);
```

Comment:
Does `terminalWrapperRef` need to be part of the dependencies? I don't think ES Lint would complain about this being removed

Reply:
Good catch!